guidelines/enhancement_template.md

```diff
@@ -272,6 +272,20 @@ to implement the design.  For instance,
  > 1. This requires exposing previously private resources which contain sensitive
   information.  Can we do this?
 
+## Security
+
+This section should cover the security impact of the change, if any. For example,
+does it introduce new privileges or capabilities? Does it expose new risks?
+
+Consider the following:
+- Which service accounts will be affected by the change, what permissions will users need?
+- Will end users need to make any security considerations when enabling this feature?
+- What is the scope of the required RBAC?
+  - Are cluster wider permissions required?
+  - Should RBAC be limited to only the namespace(s) where the feature is enabled?
+  - Should RBAC be limited to just the node(s) where the feature is enabled?
+  - Are we adding new permissions or escalation paths from Nodes to the Cluster level?
```

Comment:
Maybe we add some principles one of which should be that the security should be no worse compared to a container breakout and access to the kubelet credentials.

Reply:
Added this